nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -75,6 +75,24 @@ services.xserver.displayManager.defaultSession = "xfce+icewm";
 </programlisting>
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The testing driver implementation in NixOS is now in Python <filename>make-test-python.nix</filename>.
+     This was done by Jacek Galowicz (<link xlink:href="https://github.com/tfc">@tfc</link>), and with the
+     collaboration of Julian Stecklina (<link xlink:href="https://github.com/blitz">@blitz</link>) and
+     Jana Traue (<link xlink:href="https://github.com/jtraue">@jtraue</link>). All documentation has been updated to use this
+     testing driver, and a total of [how_many_tests_were_ported] were ported. In 20.09 the Perl driver implementation,
+     <filename>make-test.nix</filename>, is slated for removal. This should give users of the NixOS integration framework
+     a transitory period to rewrite their tests to use the Python implementation. Users of the Perl driver will see
+     this warning everytime they use it:
+<screen>
+<prompt>$ </prompt>warning: Perl VM tests are deprecated and will be removed for 20.09.
+Please update your tests to use the python test driver.
+See https://github.com/NixOS/nixpkgs/pull/71684 for details.
+</screen>
+     API compatibility is planned to be kept for at least the next release with the perl driver.
```

Comment:
I don't understand this paragraph. I mean, most if not all of the functionality from the perl test driver were ported to the python tooling (which was a quite mechanical conversion).

We might want to keep some level of compatibility for the tests written that way, but I wouldn't call that API compatibility.

Reply:
https://github.com/NixOS/nixpkgs/pull/79335#issuecomment-583660711

Reply:
Feel free to change it :+1: I think it was a bit of a mistake on my part to claim that **I** should make this PR, when it turns out I'm becoming more over extended :rofl:

Reply:
I pushed a new version. PTAL. Also, replaced the placeholder with a "hundreds of", because we don't need an exact number here.